scripts/maintenance-mode-pr.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# This script will create a PR against the dm-aws repository, toggling the maintenance mode variable as appropriate.
+#
+# Syntax: ./scripts/maintenance-mode-pr.sh [on|off] [preview|staging|production]
+
+set -e
+
+COMMAND=$(echo "${1}" | tr "[:lower:]" "[:upper:]")
+STAGE=$(echo "${2}" | tr "[:lower:]" "[:upper:]")
+STAGE_LOWER=$(echo "${STAGE}" | tr "[:upper:]" "[:lower:]")
+GIT_MAINTENANCE_MODE_BRANCH_NAME=$(date +"%Y-%m-%dT%H-%M-%S-maintenance-mode")
+
+if [ -z "${GITHUB_ACCESS_TOKEN}" ]; then
+  echo "Required environment variable GITHUB_ACCESS_TOKEN is undefined."
+  exit 1
+fi
+
+if [[ "${STAGE}" != "PREVIEW" && "${STAGE}" != "STAGING" && "${STAGE}" != "PRODUCTION" ]]; then
```

Comment:
Why not just use `STAGE_LOWER` all over the place? Save having two variables defined.

Reply:
More consistent with the other script (if we get around to generalising this), and I like having the stage name in uppercase for the PR.